central/alert/service/service_impl.go

```diff
@@ -172,8 +172,29 @@ func (s *serviceImpl) GetAlertsCounts(ctx context.Context, request *v1.GetAlerts
 	if request == nil {
 		request = &v1.GetAlertsCountsRequest{}
 	}
+
+	// TODO: @ksanchet - check correctness. This endpoint could possibly be invoked with any query.
+	// For example, alerts within a specific cluster
+	// in that case, will the code below work? The UI only sends in cluster and or namespaces but
+	// an API client can send in any query
```

Comment:
If there is a cluster, category, or severity input query then you must transform the respective query from `AddStrings` to `AddStringsHighlighted`. If not, add the wildcard.

Reply:
I had uncommitted changes I just pushed doing exactly that :-) ...check latest